src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,137 @@
+import { useEffect, useState } from 'react';
+import { Link as PersonLink, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+
+  const { slug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsLoadingError(false);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsLoadingError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const findMother = (motherName: string) => {
+    return people.find(mother => mother.name === motherName)?.slug;
+  };
+
+  const findFather = (fatherName: string) => {
+    return people.find(father => father.name === fatherName)?.slug;
+  };
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && (
+            <Loader />
+          )}
+
+          {isLoadingError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {!people.length && !isLoading && !isLoadingError && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
+
+          {people.length && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map(person => (
+                  <tr
+                    data-cy="person"
+                    className={classNames({
+                      'has-background-warning': person.slug === slug,
+                    })}
```

Comment:
do not forget about key prop

Reply:
let's also fix this 0 when table loading
<img width="493" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/70220667/2f6f8676-d8ad-4290-94fb-b3db2e7d6a37">